src/foam/u2/view/TableCellFormatter.js

```diff
@@ -121,7 +121,17 @@ foam.CLASS({
   name: 'FObjectPropertyTableCellFormatterRefinement',
   refines: 'foam.core.FObjectProperty',
 
-  properties: [ [ 'tableCellFormatter', null ] ]
+  properties: [
+    {
+      class: 'foam.u2.view.TableCellFormatter',
+      name: 'tableCellFormatter',
+      value: function(obj) {
+        this.start()
+          .add(obj.id)
```

Comment:
FObjectProperty should have a toSummary() method which should used in placed of this,. 
And this summary can overridden in the model, making this default tablecellformatter very useful.

Reply:
FObject's have toSummary() methods. So you could call obj.toSummary() instead of obj.id, and then make sure interested classes have meaningful implementations of toSummary().

Reply:
done!